go.mod

```diff
@@ -4,15 +4,15 @@ go 1.13
 
 require (
 	github.com/Masterminds/semver v1.5.0 // indirect
-	github.com/golang/protobuf v1.3.2
+	github.com/golang/protobuf v1.4.2
 	github.com/maorfr/helm-plugin-utils v0.0.0-20200216074820-36d2fcf6ae86
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/pkg/errors v0.9.1
-	github.com/spf13/cobra v0.0.5
+	github.com/spf13/cobra v0.0.7
 	github.com/spf13/pflag v1.0.5
-	helm.sh/helm/v3 v3.1.0
+	helm.sh/helm/v3 v3.1.3
```

Comment:
the latest is v3.2.4

Reply:
I did have issues updating it to v3.2.0+. When doing so, I get the following error:

```console
# github.com/maorfr/helm-plugin-utils/pkg
../../../../go/pkg/mod/github.com/maorfr/helm-plugin-utils@v0.0.0-20200216074820-36d2fcf6ae86/pkg/utils.go:69:69: not enough arguments in call to clientSet.CoreV1().Secrets(o.TillerNamespace).List
        have ("k8s.io/apimachinery/pkg/apis/meta/v1".ListOptions)
        want (context.Context, "k8s.io/apimachinery/pkg/apis/meta/v1".ListOptions)
../../../../go/pkg/mod/github.com/maorfr/helm-plugin-utils@v0.0.0-20200216074820-36d2fcf6ae86/pkg/utils.go:83:75: not enough arguments in call to clientSet.CoreV1().ConfigMaps(o.TillerNamespace).List
        have ("k8s.io/apimachinery/pkg/apis/meta/v1".ListOptions)
        want (context.Context, "k8s.io/apimachinery/pkg/apis/meta/v1".ListOptions)
../../../../go/pkg/mod/github.com/maorfr/helm-plugin-utils@v0.0.0-20200216074820-36d2fcf6ae86/pkg/utils.go:243:48: not enough arguments in call to coreV1.Pods(tillerNamespace).List
        have ("k8s.io/apimachinery/pkg/apis/meta/v1".ListOptions)
        want (context.Context, "k8s.io/apimachinery/pkg/apis/meta/v1".ListOptions)
```

I'm not too familiar with this kind of error. Could you point me in the right direction?

Reply:
Looks like helm.sh/helm/v3 3.2.x has a dependency on k8s.io/apimachinery 0.18.x, which breaks github.com/maorfr/helm-plugin-utils with a method signature that now requires a context to be passed. helm-plugin-utils is already on the latest version, so doesn't have support yet for passing in a context. Would you be comfortable to go forward with a new version depending on the latest helm 3.1.x in order to make the skip-confirmation flag available?

Reply:
sure, let's use v3.1.3, when github.com/maorfr/helm-plugin-utils supports apimachinery 0.18.x we can do another release

Reply:
also please add `skip-confirmation` flag to completion.yaml

Reply:
As @rimusz said, it is ok about updating Helm 3 until we get the utils library updated.

Reply:
Thanks to @marckhouzam for [reminding about the completion capability](https://github.com/helm/helm-2to3/pull/154#issuecomment-655091892)